.circleci/config.yml

```diff
@@ -1,831 +0,0 @@
-      # This results in shorter CI times.
-      - lint
-      - generate-docs
-      - store-docs
-      - check-dirty-git
-      - run-libmobilecoin-cbindgen
-      - check-dirty-git
-      - when:
-          condition: { equal: [ << pipeline.git.branch >>, master ] }
-          steps: [ save-sccache-cache, save-cargo-cache ]
-      - post-build
-
-  # Build/publish Android bindings
-  android-bindings:
-    executor: android-bindings-executor
-    environment:
-      <<: *default-environment
-    steps:
-      - checkout
-      - print_versions
-      - enable_sccache
-      - restore_cache:
-          name: Restore cargo cache
-          key: android-bindings-cargo-cache-stable
-      - restore_cache:
-          name: Restore sccache cache
-          key: android-bindings-sccache-cache-stable
-
-      - run:
-          name: Build
-          command: |
-            cd android-bindings
-            RUST_BACKTRACE=full make libs
-      - save_cache:
-          name: Save cargo cache
-          key: android-bindings-cargo-cache-stable-{{ epoch }}
-          paths:
-            - "/usr/local/cargo"
-      - save_cache:
-          name: Save sccache cache
-          key: android-bindings-sccache-cache-stable-{{ epoch }}
-          paths:
-            - "~/.cache/sccache"
-# FIXME: Need to install aws creds
-#      - run:
-#          name: Upload to s3
-#          command: |
-#            tar -czf android-bindings.tar.gz -C android-bindings/build .
-#            aws s3 cp android-bindings.tar.gz s3://mobilecoin.android-bindings/branches/${CIRCLE_BRANCH}-latest.tar.gz --acl public-read
-#            aws s3 cp android-bindings.tar.gz s3://mobilecoin.android-bindings/${CIRCLE_SHA1}.tar.gz --acl public-read
-#            echo
-#            echo "Bindings are available at:"
-#            echo "https://s3-us-west-1.amazonaws.com/mobilecoin.android-bindings/branches/${CIRCLE_BRANCH}-latest.tar.gz"
-#            echo "https://s3-us-west-1.amazonaws.com/mobilecoin.android-bindings/${CIRCLE_SHA1}.tar.gz"
-
-  # Build using macOS
-  build-macos:
-    parameters:
-      xcode-version: { type: string, default: *default-xcode-version }
-    executor:
-      name: macos
-      xcode-version: << parameters.xcode-version >>
-    environment:
-      <<: *default-build-environment
-      OPENSSL_BIN: /usr/local/opt/openssl/bin/openssl
-      SCCACHE_CACHE_SIZE: 450M
-      RUSTFLAGS: -D warnings -C target-cpu=penryn
-      CONSENSUS_ENCLAVE_CSS: /Users/distiller/project/sgx/css/src/valid.css
-    steps:
-      - prepare-for-build: { os: macos }
-      - run:
-          name: Cargo build
-          command: |
-            cargo build --frozen --target "$HOST_TARGET_TRIPLE" -p libmobilecoin
-      - check-dirty-git
-      - when:
-          condition: { equal: [ << pipeline.git.branch >>, master ] }
-          steps: [ save-sccache-cache, save-cargo-cache, save-homebrew-cache ]
-      - post-build
-
-  # Build and test go code
-  build-and-test-go:
```

Comment:
I guess this is where we were building and testing the go code before the move to GitHub actions

Reply:
Oh, I see I ported `lint-go` and missed this. Sorry about that!